fiboa_cli/convert_utils.py

```diff
@@ -466,3 +132,402 @@ def read_geojson(path, **kwargs):
     obj["features"] = list(map(normalize_geojson_properties, obj["features"]))
 
     return gpd.GeoDataFrame.from_features(obj, crs = "EPSG:4326")
+
+
+class BaseConverter:
+    bbox: Optional[tuple[float]] = None
+    id: str = None
+    short_name: str = None
+    title: str = None
+    license: str | dict[str, str] = None
+    attribution: str = None
+    description: str = None
+    providers: list[dict] = None
+
+    sources: Optional[dict[str, str] | str] = None
+    source_variants: Optional[dict[dict[str, str] | str]] = None
+    variant: str = None
+
+    columns: dict[str, str] = None
+    column_additions: dict[str, str] = EMPTY_DICT
+    column_filters: dict[str, callable] = EMPTY_DICT
+    column_migrations: dict[str, callable] = EMPTY_DICT
+    missing_schemas: dict[str, str] = EMPTY_DICT
```

Comment:
Out of curiosity: I've seen this recently in other code, too. Why does python need this? Why not just `var_name = {}`?

Reply:
Mutability... Now someone can not accidentally change an instance shared by many classes. You will get unexpected bugs...